src/server/worldserver/worldserver.conf.dist

```diff
@@ -2662,6 +2663,26 @@ Battleground.ReportAFK = 3
 #
 ###################################################################################################
 
+###################################################################################################
+# RATED BATTLEGROUND CONFIG
+#
+#    RatedBattleground.Enable
+#        Description: Enable or disable the rated battleground system.
+#        Default:     0 - (Disabled)
+#                     1 - (Enabled, Experimental as in not implemented)
+
+RatedBattleground.Enable = 0
+
+#
+#    RatedBattleground.Reward
+#        Description: The conquest points reward after winning a rated battleground.
+#        Default:     400
+
+RatedBattleground.Reward = 400
```

Comment:
conquest points use currency precision so I personally would prefer using 40000 instead to reflect that so we have perfect control over the value.